src/res/job_queue/queue.py

```diff
@@ -642,3 +642,30 @@ def add_ensemble_evaluator_information_to_jobs_file(
                 jobs_file.seek(0)
                 jobs_file.truncate()
                 json.dump(data, jobs_file, indent=4)
+
+    def add_experiment_server_information_to_jobs_file(
```

Comment:
Could we try to refactor the `to_jobs_file` function into a single function? Call it `add_server_information_to_jobs_file` instead, and have the `experiment_id` be optional?  I would argue we could keep the `dispatch_url` for both as well, since we are only creating one of the reporters each time anyway. 

Everything else in these two functions is identical.

Reply:
hm, can do that, just thought it would be clearer having distinct functions. But I agree in general though :)